about.html

```diff
@@ -0,0 +1,78 @@
+        <img id="egypt" src="assets/Egypt.jpg"/>
+        <img id="korea" src="assets/Korea.jpg"/>
+        <img id="jamaica" src="assets/Jamaica.jpg"/>
+
+      </section>
+      <section class="word">
+        <details>
+          <summary>PAST</summary>
+          <section>
+              <p>
+                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+              </p>
+            </section>
+        </details>
+        <details>
+          <summary>PRESENT</summary>
+          <section>
+              <p>
+                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+              </p>
+            </section>
+        </details>
+        <details>
+          <summary>FUTURE</summary>
+          <section>
+              <p>
+                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+              </p>
+            </section>
+        </details>
+        <details>
+          <summary>FUN</summary>
+          <section>
+              <p>
+                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+              </p>
+            </section>
+        </details>
+      </section>
+    </main>
+    <footer>
+      <section id="social-links" >
+        <a href="https://www.linkedin.com/in/janderoyalty/" target="_blank">
+          <img src="assets/linkedin logo.svg" />
+        </a>
+        <a href = "mailto: janderoyalty@gmail.com" target="_blank">
```

Comment:
I think this should be without a space between `mailto` and the email! it's getting flagged by the HTML validator as an error